frontend/src/views/web/our-team.html

```diff
@@ -1,125 +1,69 @@
 <div class="white-bg view-container flex-body">
     <main-header></main-header>
-    <div class="content">
-        <div class="grad-container">
-            <h4 class="team-heading">Team Awesome!</h4>
-            <div class="row">
-                <!-- integrate this card with API -->
-                <div class="col s12 l3 ">
-                    <div class="team-card">
-                        <div class="team-link-head">
-                            <span>
-                    <a href="http://deshraj.github.io/"><i class="fa fa-linkedin"></i></a>
+    <div class="grad-container">
```

Comment:
@RishabhJain2018 you have deleted `div` with class `content` from here. Please take care of these small things.
Please add it.